tools/setup

```diff
@@ -57,20 +58,30 @@ inquirer.prompt({
   const serialized_secrets = JSON.stringify(placeholder_secrets, null, '    ');
   return fs.writeFile("src/secrets.json", serialized_secrets);
 }).catch((err) => {
-  if (err) { throw err; }
+  throw err;
 }).then(() => {
+  // Let's look for zulipbot's repo in the parent directory tree.
+  return Git.Repository.open(path.normalize(__dirname + '/..'));
+}).then((r) => {
   console.log("Setting up GitHub `upstream` remote...");
   console.log("> git remote add upstream https://github.com/zulip/zulipbot.git");
-  cmd.get("git remote add upstream https://github.com/zulip/zulipbot.git", (err, data, stderr) => {
-    if (err) return console.log('An error occured!\n\n', err);
-    console.log('Done!', data);
-    console.log("Fetching from `upstream`...");
-    console.log("> git fetch upstream");
-    cmd.get("git fetch upstream", (err, data, stderr) => {
-      if (err) return console.log('An error occured!\n\n', err);
-      console.log('Done!', data);
-      console.log("Environment setup complete.\n");
-      console.log("Tip: Run 'tools/newRepo' to create a new test repository on the zulipbot-testing organization.");
-    });
-  });
+  repo = r;
```

Comment:
Because doing that would limit `repo`'s scope tho the anonymous function, and we need that variable for the next step of the promise chain.

Can't declare it as a `const` because the initializaton isn't where it was declared.